be/src/olap/rowset/rowset_reader_context.h

```diff
@@ -45,6 +46,7 @@ struct RowsetReaderContext {
     // column name -> column predicate
     // adding column_name for predicate to make use of column selectivity
     const std::vector<ColumnPredicate*>* predicates = nullptr;
+    const std::vector<ColumnPredicate*>* all_compound_predicates = nullptr;
```

Comment:
a more intuitive name

Reply:
rename to `predicates_except_leafnode_of_andnode`